### DISCUSSION TITLE
Videos and a discord?

### DISCUSSION THREAD

**Discussion by Grey03** [TomSchimansky/CustomTkinter#1031] [Ideas]
Absolutely loving customtkinter! I use it all the time, currently I'm working on a project and it seems like videos will be a bit of a pain to get working, will they be built in at some point or will we just have to work around? Also do you have some sort of discord or forum other than here to discuss customtkinter?

**Comment by Akascape** [comment]
@Grey03 You can use a module called [tkvideoplayer](https://pypi.org/project/tkvideoplayer/) with customtkinter to play video files.
An example is given here: https://github.com/TomSchimansky/CustomTkinter/discussions/531
